pkg/agent/openflow/client_test.go

```diff
@@ -65,6 +65,8 @@ var (
 
 	fakeL7NPTargetOFPort = uint32(10)
 	fakeL7NPReturnOFPort = uint32(11)
+
+	groupIDAllocator = NewGroupAllocator()
```

Comment:
Declaring it here means once another test uses it, the group ID it gets will be unpredicatable.

Reply:
Changed "groupIDAllocator" as a local variable when creating fakeClient.